lib/process-services-cloud/src/lib/people/components/people-cloud.component.ts

```diff
@@ -529,6 +556,22 @@ export class PeopleCloudComponent implements OnInit, OnChanges, OnDestroy {
         this.searchUsers$.next(this._searchUsers);
     }
 
+    private isUserBelongToRestrictedGroups(user: IdentityUserModel): Observable<boolean> {
+        return this.getUserGroups(user.id).pipe(
+            map(userGroups => userGroups.filter(
+                group => this.groupsRestriction.filter(
```

Comment:
you can probably use this.groupRestriction.includes(group.name) instead of iterating and filtering

Reply:
change to includes instead of filtering